### PR TITLE
Added build arg to instarr BDB and OSM dev plugins

### PR DESCRIPTION
**Pull request by luipir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ cd docker-geogig
 development build or to a specific version, the default
 value is `1.0-RC3` and it will build geogig-`1.0-RC3`.
 
+**BDBPLUGIN** build arg to be se set to BDB to install Berkley DB dev geogig plugin
```
Comment (timlinux) [inline README.md]: se set -> set

File: README.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ cd docker-geogig
...
+
+**OSMPLUGIN** build arg to be se set to OSM to install OSM dev geogig plugin
```
Comment (timlinux) [inline README.md]: se set -> set

**Review by timlinux — CHANGES_REQUESTED** [review]

**Comment by timlinux** [comment]
Thanks for the patch @luipir !

**Comment by luipir** [comment]
done changes to readme

**Pull request merged**